### PR TITLE
Update advanced-hunting-devicenetworkevents-table.md

### LINKED ISSUE #11683: Description update request for "LocalIP"
[Enter feedback here]
Local IP Address and Remote IP address displayed on the AH result sometimes opposite with docs description.

According to the doc's description, "LocalIP" should be the IP assigned to the local machine.

LocalIP | string | IP address assigned to the local machine used during communication
RemoteIP | string | IP address that was being connected to

But querying DeviceNetworkEvents depends on the specific communication, result shows Locally assigned IP as Remote IP.

image

This was escalated and concluded that it's came from enhancement of device discovery.
https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/hunting-for-network-signatures-in-microsoft-defender-for/ba-p/3429520

So now the description of "LocalIP" explanation is inconsistent.

This is just my idea but please consider to update the description like below.
Now: "IP address assigned to the local machine used during communication"
After:  "Source IP address which is attempting to communicate from"

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3d4a0076-9941-2d8d-aa0d-4ddd7a74c09a
* Version Independent ID: c14a1d91-a692-4d19-7796-b5eca1341f4d
* Content: [DeviceNetworkEvents table in the advanced hunting schema](https://learn.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-DeviceNetworkEvents-table?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender/advanced-hunting-devicenetworkevents-table.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/advanced-hunting-devicenetworkevents-table.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @schmurky
* Microsoft Alias: **maccruz**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to Local IP description

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11683

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/security/defender/advanced-hunting-devicenetworkevents-table.md
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/advanced-hunting-devicenetworkevents-table.md]:
```suggestion
| `LocalIP` | `string` | Source IP address from which communication is attempted |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Comment by schmurky** [comment]
I am checking with my SME whether this is accurate. They are coming back next week from OOF. I will update ASAP.

**Review by schmurky — APPROVED** [review]
can you tweak a little to say: "Source IP, or the IP address where the communication came from" as this was the language approved @msbemba @yogkumgit, after that it is good to go, can you do the merge yourself, or should it be me (I don't work on the public repo so apologies if that is the case)?

File: microsoft-365/security/defender/advanced-hunting-devicenetworkevents-table.md
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/advanced-hunting-devicenetworkevents-table.md]:
```suggestion
| `LocalIP` | `string` | Source IP, or the IP address where the communication came from |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, please commit the suggested change as per @schmurky's suggestion. Thanks.

**Pull request merged**
